.travis.yml

```diff
@@ -8,6 +8,9 @@ cache:
   - depends/built
   - depends/sdk-sources
   - $HOME/.ccache
+stages:
+  - check_doc
+  - test
 env:
   global:
     - MAKEJOBS=-j3
```

Comment:
Can remove the CHECK_DOC on the line below this one.

Reply:
Done